internal/wal/gen_wal_proto.sh

```diff
@@ -3,12 +3,26 @@
 # Exit immediately if a command exits with a non-zero status
 set -e
 
-# Define the directory where the proto file resides
+# Define the directory where the proto file will be downloaded
 PROTO_DIR=$(dirname "$0")
+OUTPUT_DIR="$PROTO_DIR"
 
-# Define the proto file and the output file
+# Define the GitHub repository and file path
+GITHUB_REPO="AshwinKul28/dicedb-protos"
+GITHUB_BRANCH="patch-1"
+PROTO_FILE_PATH="wal/wal.proto"
 PROTO_FILE="$PROTO_DIR/wal.proto"
-OUTPUT_DIR="."
+
```

Comment:
OUr protos dependency is taken through dicedb-go package which takes it from dicedb-protos. We need not be doing this here. The generates WAL protos can be part of dicedb-go.